great_expectations/data_context/data_context.py

```diff
@@ -282,6 +283,15 @@ def __init__(self, project_config, context_root_dir=None, runtime_environment=No
     def _build_store(self, store_name, store_config):
         module_name = "great_expectations.data_context.store"
         try:
+            # Set expectations_store.store_backend_id to given data_context_id if it doesnt exist
+            if (store_name == self.expectations_store_name) and store_config.get(
+                "store_backend"
+            ):
+                store_config["store_backend"].update(
+                    {
+                        "manually_initialize_store_backend_id": self.anonymous_usage_statistics.data_context_id
```

Comment:
Can you clarify what "manually_initialize_store_backend_id" does? I don't follow yet.

Reply:
Sure! From the docstring in [StoreBackend.store_backend_id()](https://github.com/great-expectations/great_expectations/pull/2046/files#diff-3e42f11c8884077bdeff67d7316330c4f63a792505d8019629dde7dd39af2a07R43)
```
        If a valid UUID store_backend_id is passed in param manually_initialize_store_backend_id
        and there is not already an existing store_backend_id then the store_backend_id
        from param manually_initialize_store_backend_id is used to create it.
```
This is so we can manually set the store_backend_id, e.g. when the env var GE_DATA_CONTEXT_ID is set like in [this test](https://github.com/great-expectations/great_expectations/pull/2046/files#diff-c3b5b880409a33e00a2a5f1bb2655ba5cf9e14ec6ec92ced6237a0907f4634faR295).

I'm also very open to renaming it to something more descriptive!

Reply:
There is a possibility this is not necessary, looking into it.

Reply:
My mistake - I forgot that I'm using the `manually_initialize_store_backend_id` parameter to set the `expectations_store.store_backend_id` to the `project_config.anonymous_usage_statistics.data_context_id` if the expectations_store doesn't yet exist